src/lib/pages/backoffice/EItem/EItemDetails/File/state/actions.js

```diff
@@ -28,6 +33,12 @@ export const uploadFile = (eitemPid, bucket, file) => {
         type: SUCCESS,
         payload: response.data,
       });
+      dispatch(
+        sendSuccessNotification(
+          'Success!',
+          'The file needs some seconds to be displayed to the frontsite.'
```

Comment:
Very very minor and just IMHO (for other reviewers, feel free to comment):
```suggestion
          'The uploaded file will be available to users in a few seconds.'
```

Reply:
👍